rust/oak/src/storage/mod.rs

```diff
@@ -76,6 +76,9 @@ impl Storage {
         );
 
         let mut request_any = protobuf::well_known_types::Any::new();
+        request_any.set_type_url(
```

Comment:
Should we fix this upstream perhaps? Maybe open an issue in the corresponding repo and link it here.

Reply:
Done.